### PR TITLE
DM-34683: Import Instrument class from lsst.pipe.base

### PR DESCRIPTION
**Pull request by mxk62**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,6 @@
 ######################
 dbb_buffmngrs_endpoint
 ######################
+
+Ingests files into different data management systems operating on the
+endpoint site, e.g., Gen2 and/or Gen3 Butler repositories.
```
Comment (timj) [inline README.rst:6]: IS the gen2 code still active?

**Review by timj — APPROVED** [review]

**Comment by timj** [comment]
There is a typo in the commit summary: "iingestig"

File: README.rst (hunk 1 above)
Comment (mxk62) [reply README.rst:6]: If by active you meant "in use" then the answer is yes. I was asked to keep automatic ingest services to Gen2 repos in NCSA up and running even after Gen2 Butler was officially retired.

Comment (timj) [reply README.rst:6]: There are no gen2 LATISS or ComCam users left as far as I know 😄

Comment (mxk62) [reply README.rst:6]: I'll relay this information to Steve, but I'm not pulling the plug on them on my own. :wink:

**Pull request merged**